packages/next/src/client/components/react-dev-overlay/server/middleware-turbopack.ts

```diff
@@ -96,27 +96,24 @@ export async function createOriginalStackFrame(
             },
             { forceColor: true }
           ),
-  }
-}
-
-function stackFrameFromQuery(query: ParsedUrlQuery): TurbopackStackFrame {
-  return {
-    file: query.file as string,
-    methodName: query.methodName as string,
-    line:
-      typeof query.lineNumber === 'string' ? parseInt(query.lineNumber, 10) : 0,
-    column:
-      typeof query.column === 'string' ? parseInt(query.column, 10) : null,
-    isServer: query.isServer === 'true',
+    sourcePackage,
   }
 }
 
 export function getOverlayMiddleware(project: Project) {
   return async function (req: IncomingMessage, res: ServerResponse) {
-    const { pathname, query } = url.parse(req.url!, true)
+    const { pathname, searchParams } = new URL(req.url!, 'http://n')
+
+    const frame = {
```

Comment:
unsure why this was inlined, doesn't really matter though

Reply:
Dropped the `url.parse` usage in favor of the Web standard `URL`. Also, parsing the search params has previously been done in two locations so it made sense to have an extracted function, but it could have been done just once higher up.